docs/introduction/Examples.md

```diff
@@ -1 +1,137 @@
-## Examples
\ No newline at end of file
+## Examples
+### Simple - reading from an existing xlsx spread sheet.
+The following C plus plus code will read the values from an xlsx file and print the string values to the screen. This is a very simple example to get you started.
+
+```
+#include <iostream>
+#include <xlnt/xlnt.hpp>
+
+using namespace std;
+using namespace xlnt;
+int main()
+{
+    workbook wb;
+    wb.load("/home/timothymccallum/test.xlsx");
+    auto ws = wb.active_sheet();
+    clog << "Processing spread sheet" << endl;
+    for (auto row : ws.rows(false)) 
+	{ 
```

Comment:
Just to be consistent with the rest of xlnt, I think it would be good to keep the indent level of opening braces on the same level as the preceding statement (i.e. Allman-style indentation).